kahuna/public/js/services/api/collections-api.js

```diff
@@ -127,6 +127,35 @@ collectionsApi.factory('collections',
             });
     }
 
+    function findCollectionResource(image, collectionToMatch){
+        return image.data.collections.filter(collection => {
+            return collection.data.pathId === collectionToMatch;
+        });
+    }
+
+    function getCollectionToRemove(image, collection) {
+        const filteredCollections = findCollectionResource(image, collection);
```

Comment:
[If you use a find](https://github.com/guardian/grid/pull/1686/files#r50559228), the name could change too.